test/verify/check-networkmanager-basic

```diff
@@ -165,6 +171,14 @@ class TestNetworkingBasic(NetworkCase):
         assert_running()
         wait(lambda: m.execute("systemctl is-enabled NetworkManager"))
 
+        # /usr in coreos is readonly
+        if m.image not in ["fedora-coreos"]:
```

Comment:
I would have used `if m.ostree_image:` or `m.needs_writable_usr()`, but we can clean this up if/when it gets in the way.